services/wfm/src/main/java/org/openkilda/wfm/topology/nbworker/bolts/IslNotFoundException.java

```diff
@@ -0,0 +1,22 @@
+/* Copyright 2018 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.wfm.topology.nbworker.bolts;
+
+public class IslNotFoundException extends RuntimeException {
```

Comment:
I think `IslNotFoundException` should be checked exception to be 100% sure that we will handle it somewhere. 
RuntimeException may cause 500 InternalServerError it we will forget to handle it.